float64 dist
float64 theta
---------------
bool res